modules/nodes-nodes-working-setting-booleans.adoc

```diff
@@ -0,0 +1,56 @@
+[id="nodes-nodes-working-setting-booleans"]
+
+= Setting SELinux booleans
+
+{product-title} allows you to enable and disable a SELinux boolean on a RHEL CoreOC node. The following procedure explains how to modify SELinux booleans on nodes using the MachineConfigOperator (MCO). The `container_manage_cgroup` is the example sebool in this procedure. You can modify this value to whichever sebool you need.
```

Comment:
s/ MachineConfigOperator / Machine Config Operator (<- that's how it's typically written in the docs. I couldn't find MachineConfigOperator anywhere)